relayer/relays/beacon/state/beacon.go

```diff
@@ -259,6 +339,24 @@ type ExecutionPayload struct {
 	Transactions  [][]byte  `ssz-max:"1048576,1073741824" ssz-size:"?,?" json:"transactions"`
 }
 
+type ExecutionPayloadCapella struct {
+	ParentHash    [32]byte      `ssz-size:"32" json:"parent_hash"`
+	FeeRecipient  [20]byte      `ssz-size:"20" json:"fee_recipient"`
+	StateRoot     [32]byte      `ssz-size:"32" json:"state_root"`
+	ReceiptsRoot  [32]byte      `ssz-size:"32" json:"receipts_root"`
+	LogsBloom     [256]byte     `ssz-size:"256" json:"logs_bloom"`
+	PrevRandao    [32]byte      `ssz-size:"32" json:"prev_randao"`
+	BlockNumber   uint64        `json:"block_number"`
+	GasLimit      uint64        `json:"gas_limit"`
+	GasUsed       uint64        `json:"gas_used"`
+	Timestamp     uint64        `json:"timestamp"`
+	ExtraData     []byte        `ssz-max:"32" json:"extra_data"`
+	BaseFeePerGas [32]byte      `ssz-size:"32" json:"base_fee_per_gas"`
+	BlockHash     [32]byte      `ssz-size:"32" json:"block_hash"`
+	Transactions  [][]byte      `ssz-max:"1048576,1073741824" ssz-size:"?,?" json:"transactions"`
+	Withdrawals   []*Withdrawal `ssz-max:"16" json:"withdrawals"`
```

Comment:
Similarly as with the `BeaconStateCapellaMainnet` struct, if `Withdrawals` is the only new field we can reuse the `ExecutionPayloadBellatrix` struct and only add the new `Withdrawals` field.